cypress/support/fragments/settings/inventory/loanTypes.js

```diff
@@ -0,0 +1,16 @@
+import { including, Pane, HTML, PaneHeader, NavListItem } from '../../../../../interactors';
```

Comment:
Please, move file to cypress/support/fragments/settings/inventory/items/loanTypes.js